pkg/profile/artifact.go

```diff
@@ -46,28 +47,35 @@ type MakerConfig struct {
 // ProfilesArtifactsMaker creates a list of artifacts from profiles data.
 type ProfilesArtifactsMaker struct {
 	MakerConfig
+
+	Builders     map[int]Builder
+	nestedName   string
+	profileRepos []string
 }
 
 // NewProfilesArtifactsMaker creates a new profiles artifacts maker.
 func NewProfilesArtifactsMaker(cfg MakerConfig) *ProfilesArtifactsMaker {
+	builders := map[int]Builder{
+		KUSTOMIZE: &KustomizeBuilder{
```

Comment:
Yeah this could be some kind of shared config, like builder config. I haven't gotten that far yet. :)